Gemfile

```diff
@@ -113,7 +113,6 @@ end
 group :test do
   gem 'axe-core-rspec', '~> 4.2'
   gem 'bundler-audit', require: false
-  gem 'capybara-selenium', '>= 0.0.6'
```

Comment:
this is just because we no longer use selenium, right?

Reply:
the `webdrivers` gem contains all of the different webdrivers,  and `capybara-selenium` hasn't been updated since 2014. This gem was just there to make capybara work with selenium, but that is handled with webdrivers now.

Reply:
Excellent! ✨ really glad to get that outdated gem outta there